encodings/fastlanes/benches/canonicalize_bench.rs

```diff
@@ -105,8 +105,10 @@ fn canonical_into<T: NativePType>(bencher: Bencher, (arr_len, chunk_count): (usi
         .into_array();
 
     bencher.bench(|| {
-        let mut primitive_builder =
-            PrimitiveBuilder::<T>::with_capacity(arr.dtype().nullability(), arr_len * chunk_count);
+        let mut primitive_builder = PrimitiveBuilder::<T>::with_capacity(
+            arr.dtype().nullability(),
+            arr_len * chunk_count + 1024,
+        );
         chunked
```

Comment:
I know this was me, but we need to remove the clone from the below benchmarks e.g.
```rust 
    bencher
        .with_inputs(|| chunked.clone())
        .bench_local_values(|chunked| chunked.into_canonical().vortex_unwrap().len());
```